specutils/tests/test_spectrum1d.py

```diff
@@ -110,3 +111,18 @@ def test_create_with_uncertainty():
 
     assert spec.flux.unit == spec.uncertainty.unit
 
+
+def test_read_linear_solution():
+    file_path = os.path.join(os.path.dirname(__file__),
+                             'data/L5g_0355+11_Cruz09.fits')
+
+    spec = Spectrum1D.read(file_path, format='wcs1d-fits')
+
+    assert isinstance(spec, Spectrum1D)
+
+    assert isinstance(spec.flux, u.Quantity)
+    assert isinstance(spec.wavelength, u.Quantity)
```

Comment:
I think we should check this with `spec.spectral_axis`, because using `.wavelength`, `.frequency`, `.energy` will attempt to put the dispersion solution through astropy quantities, which itself should be a different test.

(I say this because there's no `spectral_axis_unit` in the test data, so `spec.wavelength` fails because it cannot convert to a length physical type.)